Custom_GruppeAdler/createVehicle.sqf

```diff
@@ -0,0 +1,70 @@
+createrebelVehicle = {
+	_pos = _this select 0;
+	_vehicleType = _this select 1;
+	_side = _this select 2;
+
+	_crewCount = floor (random 1);
+
+
+	_createDriver = {
+		_grp0 = _this select 0;
+		_car0 = _this select 1;
+
+		_crewDriver = _grp0 createUnit ["C_man_1", _pos, [], 0, "CARGO"];
+
+		[_crewDriver] call randomCivilian; // rebel loadout
+		[_crewDriver, _car0, true] call ACE_VehicleLock_fnc_addKeyForVehicle;
```

Comment:
gives the driver a key for the car - but car is *locked* initially, so driver has to get out and ripped to drive it.